tests/src/Interop/COM/NativeServer/StringTesting.h

```diff
@@ -312,6 +312,23 @@ class StringTesting : public UnknownImpl, public IStringTesting
         return S_OK;
     }
 
+    DEF_FUNC(Reverse_LPWSTR_With_LCID)(
+        /*[in]*/ LPWSTR a,
+        /*[in]*/ LCID lcid,
```

Comment:
Can you comment that the `lcid` is purely verifying the a placeholder for the argument was inserted? I assume that was the intent here.